src/React.Core/ReactSiteConfiguration.cs

```diff
@@ -216,17 +216,19 @@ public IReactSiteConfiguration SetMaxUsagesPerEngine(int? maxUsagesPerEngine)
 		}
 
 		/// <summary>
-		/// Gets or sets whether the MSIE engine should be used if V8 is unavailable.
+		/// Gets or sets whether to allow the JavaScript pre-compilation (accelerates the
+		/// initialization of JavaScript engines).
 		/// </summary>
-		public bool AllowMsieEngine { get; set; }
+		public bool AllowJavaScriptPrecompilation { get; set; }
```

Comment:
This is a breaking change (removing a field from the public API) so we need to ensure we do a major version bump for this. However, I think we are already bumping the major version due to using the latest beta JavaScriptEngineSwitcher, so it should be fine.

Reply:
`AllowMsieEngine` property was removed from the `IReactSiteConfiguration` interface in the [“Remove auto-registered JS engines from React.Core”](https://github.com/reactjs/React.NET/pull/582) PR. I just removed the unused code.

Reply:
I plan to release a new version of the JavaScript Engine Switcher (Release Candidate 1) in next few days.